### ISSUE TITLE
Unable to `go install` with `v0.7.0` & `latest` revisions

### ISSUE THREAD

**Issue by nocive** [jwilder/dockerize#195]
go install 0.7.0 (fails)
```
# go install github.com/jwilder/dockerize@v0.7.0
go: downloading github.com/jwilder/dockerize v0.7.0
go: github.com/jwilder/dockerize@v0.7.0: github.com/jwilder/dockerize@v0.7.0: parsing go.mod:
	module declares its path as: dockerize
	        but was required as: github.com/jwilder/dockerize
```

go install latest (fails)
```
# go install github.com/jwilder/dockerize@latest
go: github.com/jwilder/dockerize@latest: github.com/jwilder/dockerize@v0.7.0: parsing go.mod:
	module declares its path as: dockerize
	        but was required as: github.com/jwilder/dockerize
```

go install 0.6.0 (succeeds)
```
# go install github.com/jwilder/dockerize@v0.6.0
go: downloading github.com/jwilder/dockerize v0.6.0
go: finding module for package golang.org/x/net/context
go: finding module for package github.com/jwilder/gojq
go: finding module for package github.com/hpcloud/tail
go: downloading golang.org/x/net v0.12.0
go: downloading github.com/hpcloud/tail v1.0.0
go: downloading github.com/jwilder/gojq v0.0.0-20161018055142-c550732d4a52
go: found github.com/hpcloud/tail in github.com/hpcloud/tail v1.0.0
go: found github.com/jwilder/gojq in github.com/jwilder/gojq v0.0.0-20161018055142-c550732d4a52
go: found golang.org/x/net/context in golang.org/x/net v0.12.0
go: finding module for package gopkg.in/fsnotify.v1
go: finding module for package github.com/elgs/gosplitargs
go: finding module for package gopkg.in/tomb.v1
go: downloading gopkg.in/fsnotify.v1 v1.4.7
go: downloading gopkg.in/tomb.v1 v1.0.0-20141024135613-dd632973f1e7
go: downloading github.com/elgs/gosplitargs v0.0.0-20230310130726-7d16e488436a
go: found gopkg.in/tomb.v1 in gopkg.in/tomb.v1 v1.0.0-20141024135613-dd632973f1e7
go: found github.com/elgs/gosplitargs in github.com/elgs/gosplitargs v0.0.0-20230310130726-7d16e488436a
go: found gopkg.in/fsnotify.v1 in gopkg.in/fsnotify.v1 v1.4.7
go: downloading golang.org/x/sys v0.10.0
```

**Comment by nocive** [comment]
I noticed now that this is already fixed by https://github.com/jwilder/dockerize/commit/11dc70058198aa6772afc1155a731c148f0a9b2a.

@jwilder any chance you can release a patched version soonish? :pray:

**Comment by jwilder** [comment]
Should be fixed w/ v0.9.1
